engine/class_modules/sc_death_knight.cpp

```diff
@@ -4898,7 +4905,7 @@ struct frostscythe_t : public death_knight_melee_attack_t
     }
 
     // Frostscythe procs rime at half the chance of Obliterate
-    p() -> buffs.rime -> trigger( 1, buff_t::DEFAULT_VALUE(), p() -> spec.rime -> effectN( 2 ).percent() / 2.0 );
+    p() -> buffs.rime -> trigger( 1, buff_t::DEFAULT_VALUE(), rime_proc_chance );
```

Comment:
Something I didn't think about when originally writing this, you could use ``buffs.rime -> manual_chance / 2`` instead of storing the rime proc chance value or calculating it from spelldata.